### DISCUSSION TITLE
TerriaMap with v8 doesn't work

### DISCUSSION THREAD

**Discussion by sammyaxe** [TerriaJS/terriajs#5230] [Q&A] [answered]
Hi there, 

I'm struggling whole day to get v8 working with TerriaMap, is TerriaMap intended to be used with v8? 

changing version in package.json to
`  "terriajs": "8.0.0-alpha.65",`

running npm install work but it fails at npm run gulp build.
running yarn install and run gulp build compiles the files but running npm start leads to blank page

Any suggestions would be welcome at this point.

**Comment by nf-s** [comment, marked as answer]
Hi @digiiitalmb 

Which TerriaMap branch are you using?
The [`TerriaMap - next`](https://github.com/TerriaJS/TerriaMap/tree/next) branch should work.

There may be some issues with `npm run start` (with `pm2`), but this command should work to start TerriaJS-server:

```
node ./node_modules/terriajs-server/lib/app.js --config-file devserverconfig.json
```

Sorry for the v8 issues! We are working to improve docs as we speak.

**Comment by nf-s** [comment]
The TerriaMap next branch uses `terriajs:8.0.0-alpha.64`, but it will also work with `terriajs:8.0.0-alpha.65`

**Comment by KeyboardSounds** [comment]
We also need to update terriajs-cesium for a new default ion token so that the Bing basemaps and terrain appear. Otherwise the globe will be plain blue. (but this isn't v8 specific)

**Comment by sammyaxe** [comment]
thanks for the help, using "next"branch did the tricks
